### DISCUSSION TITLE
AWS region

### DISCUSSION THREAD

**Discussion by tmt-devq** [supabase/supabase#2949] [Questions] [answered]
Hi,

I set up a project during the alpha stage of Supabase, and was able to choose South Africa as a region:

<img width="877" alt="image" src="https://user-images.githubusercontent.com/89100093/130738360-fa86111d-e980-4dcc-ae43-106688aa2bfb.png">

When I tried to create a new project this morning, I see it's no longer available as an option:

<img width="682" alt="image" src="https://user-images.githubusercontent.com/89100093/130738566-97b1229a-bda2-47d2-bca5-9cb1dfd622d8.png">

Any chance of getting that added back?
Thanks

**Comment by darora** [comment, marked as answer]
Hey @tmt-devq - my apologies, but we're not supporting afs1 for new projects for now as the region requires a lot of special casing that increases our operational complexity and burden.

I'd recommend trying out https://ping.psa.fun/ or something similar to pick the next closest region.

**Reply by tmt-devq** [reply]
Thank you for the feedback @darora . Its a real pity, the latency jumps from 38ms to 177ms for the second best, so that's quite a hit.

**Reply by AndreBooysen** [reply]
Hi @tmt-devq  , is there any chance that AWS has resolved or added the features you need in the afs1 region by now. It will be much appreciated  if you can get the South African region live. As @darora  stated, our closest hop from here is Europe and that is a full continent away.

**Reply by chris-snow** [reply]
@darora Are there any plans to bring back afs1 or is it shelved for the foreseeable future?

**Reply by marcoMendes321** [reply]
@darora any plans to bring afs1 back? Any updates on future plans would be greatly appreciated🙏

**Reply by Thabhelo** [reply]
@darora Are there any plans to bring back afs1? Why only us?

**Comment by rodbs** [comment]
Are you planning to support more AWS regions?  (mostly in Europe - today AWS Spain opened for business)
